options.rb

```diff
@@ -0,0 +1,139 @@
+require './student_class'
+require './teacher_class'
+require './book'
+require './rental'
+require './input'
+
+# options class
+class Options
+  def initialize
+    @books = []
+    @persons = []
+    @rentals = []
+    @input = Input.new
+  end
+
+  def options
+    puts
+    puts 'Please choose an option by enterin a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+  end
+
+  def options_cases(user_input)
+    case user_input
+    when '1'
+      list_books
+    when '2'
+      list_people
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      list_all_rental_by_id
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    student_or_teacher = @input.read
+    case student_or_teacher
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'Wrong Input!'
+      return
+    end
+    puts 'Person created successfully'
+  end
+
+  def create_teacher
+    print 'Age: '
+    age = @input.read
+    print 'Name: '
+    name = @input.read
+    print 'Specialization: '
+    specialization = @input.read
+    @persons.push(Teacher.new(specialization: specialization, age: age.to_i, name: name))
+  end
+
+  def create_student
+    print 'Age: '
+    age = @input.read
+    print 'Name: '
+    name = @input.read
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = @input.read
+    unless parent_permission.upcase == 'Y' || parent_permission.upcase == 'N'
+      puts 'Wrong Input!'
+      return
+    end
+    parent_permission = parent_permission.upcase == 'Y'
+    @persons.push(Student.new(age: age.to_i, name: name, parent_permission: parent_permission))
+  end
+
+  def create_book
+    print 'Title: '
+    title = @input.read
+    print 'Author: '
+    author = @input.read
+    @books.push(Book.new(title, author))
+    puts 'Book created successfully'
+  end
```

Comment:
- [ ] in this method you are printing to the console getting information from the user and saving the information this is not in line with the single responsibility rule, have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) for more information.